src/main/java/de/qabel/desktop/ui/remotefs/RemoteFSController.java

```diff
@@ -257,6 +257,7 @@ private void setCellValueFactories() {
     }
 
     private Callback<TreeTableColumn.CellDataFeatures<BoxObject, Node>, ObservableValue<Node>> inlineOptionsCellValueFactory() {
+        Map<String, Observer> observers = new WeakHashMap<String, Observer>();
```

Comment:
And I added this cache to the nav observer because the TreeTableView creates a lot of columns for the same node ... and each of these cells kept sticking to the nav and was never removed. With the sync activated, I had hundrets of observers sticking around just by syncing 5 or 6 files.